main.py

```diff
@@ -233,15 +232,18 @@ def create_circuit(self, data):
                 evc.deploy()
 
         # Notify users
-        event = KytosEvent(
-            name="kytos.mef_eline.created", content=evc.as_dict()
-        )
-        self.controller.buffers.app.put(event)
-
         result = {"circuit_id": evc.id}
         status = 201
         log.debug("create_circuit result %s %s", result, status)
-        emit_event(self.controller, "created", evc_id=evc.id)
+        emit_event(self.controller, name="created", content={
+            "evc.id": evc.id,
```

Comment:
Aldo, let's introduce a helper function `map_evc_event_content(evc)` or `build_evc_event_content(evc)` or something similar on utils.py that takes and evc and returns a dict of the event content. That way if we need to augment this the change will be in a single place later on since we know upfront that the content of these events are expected to be the same.

Reply:
That way we also standardize the ID being sent with either `"evc_id"` or `"evc.id"` (seems that `"evc_id" is more idiomatic in Python land), facilitating for callers when accessing the evc id of these events using the same key.

Reply:
Function `map_evc_event_content(evc)` added on commit [8d2868713e2d0be7202b97791df6be09ce5dd8c2](https://github.com/kytos-ng/mef_eline/pull/257/commits/8d2868713e2d0be7202b97791df6be09ce5dd8c2)

Reply:
Nicely done. Let me close this thread.